client/webserver/site/src/js/markets.ts

```diff
@@ -1084,7 +1086,7 @@ export default class MarketsPage extends BasePage {
     if (!this.maxLoaded) this.maxLoaded = app().loading(page.maxOrd)
     const [bid, qid] = [this.market.base.id, this.market.quote.id]
     const [bWallet, qWallet] = [app().assets[bid].wallet, app().assets[qid].wallet]
-    if (!bWallet || !bWallet.running || !qWallet || !qWallet.running) return
+    if (!bWallet || !bWallet.running || !qWallet || !qWallet.running || !this.market.dex) return
```

Comment:
Is it possible to have `this.market` but not `this.market.dex`? I don't think it is.

Reply:
> Is it possible to have `this.market` but not `this.market.dex`? I don't think it is.

I had the _exact_ same question.  This is really foggy.

Reply:
@ukane-philemon Unless you see a way that I don't, we can assume that if `this.market` is not `undefined`, then `this.market.dex` is also populated. That means you can remove the chaining operator after `dex` throughout.

Reply:
I think I missed this, will remove.